src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1108,6 +1120,18 @@ private static ObjectType GetParseCidrReturnType()
             }, null);
         }
 
+        private static ObjectType GetUriReturnType()
```

Comment:
```suggestion
        private static ObjectType GetParseUriReturnType()
```

Nit: There is [a different, unrelated `uri` function](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-string#uri), so it's better to use the full function name for clarity